__tests__/app.test.js

```diff
@@ -76,4 +76,44 @@ describe("GET requests", () => {
           });
       });
   });
+
+
+  describe("GET /api/reviews/:review_id", () => {
+    test("Status: 200, given an id returns a matching review object", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({ body }) => {
+          const { review } = body;
+            expect(review).toEqual({
```

Comment:
Consider using a looser jest matcher here. If a new column gets added to our database in the future this test would break, we want to avoid ever having to go back to edit old tests.